user_interaction/static/themes/dark-theme.css

```diff
@@ -40,13 +40,34 @@
     --greenify-dropdown-hover-text-color: inherit;
     --greenify-dropdown-active-bg-color: var(--greenify-bg-color);
     --greenify-dropdown-active-text-color: var(--greenify-text-color);
+
+    /*Calendar day color*/
+    --fc-neutral-bg-color: green;
```

Comment:
I can't seem to find what this actually changes on the calendar page

Reply:
In the calendar list function, makes day green instead of white
New
![image](https://user-images.githubusercontent.com/75546808/144091717-f12cb2c4-637a-4889-b5f2-222c1f243019.png)

before
![image](https://user-images.githubusercontent.com/75546808/144091876-f72912b6-8721-42e8-8eb2-87b579635442.png)